amundsen_application/static/js/components/common/ResourceListItem/styles.scss

```diff
@@ -85,4 +85,8 @@
   .title-2 {
     color: $resource-title-color;
   }
+
+  .underline {
+    border-bottom: solid #9191A8;
```

Comment:
Something that is missing on the styling is the gray background that is supposed to show up on hover. I notice in Figma the styles for the rectangle are as follows:
<img src='https://user-images.githubusercontent.com/1790900/83477495-34e14300-a448-11ea-891a-f5478660682a.png' width='30%' />

So to achieve this we could change this to:
```
.underline {
    box-shadow: 0px 1px 0px $gray40; // can also make a new variable for this

    &:hover,
    &:focus {
      background: $btn-default-bg-hover;
    }
  }
```

Lastly, this file is appropriate for styles that will be shared across the components, but since this style is specific to `SchemaInfo`, I would actually create a styles file for `SchemaInfo` and put it there instead.